npm-shrinkwrap.json

```diff
@@ -127,7 +133,7 @@
     },
     "angular-if-state": {
       "version": "1.0.0",
-      "from": "angular-if-state@>=1.0.0 <2.0.0",
+      "from": "angular-if-state@1.0.0",
```

Comment:
Travis is getting "upset" with so many changes in `npm-shrinkwrap.json` :-/  Can you check you're using the 'right' version of NPM?

Reply:
(and ping @jviotti too, in case he remembers how we resolved this problem the last time it cropped up)

Reply:
I was using the "right" npm version (which is node 6 with npm 3.10.x), the issue with this that the from field depends on the previous recorded version of the package, and will inevitably change when updating dependencies.

Reply:
:cry: 
But even so, why is the shrinkwrap file generated on Travis different to the shrinkwrap file in this PR? Or is it a horrible order-of-installation problem?

Reply:
So, I've just triple-checked – I'm on the same versions of node & npm as the travis box is, I shrinkwrapped multiple times, and no change – `make sanity-checks` comes back clean.

Reply:
CI servers seem happy now (and the shrinkwrap diff is much smaller, which is nice)
:-)